lib/Command/Configure.php

```diff
@@ -108,8 +109,10 @@ private function configureSignal(InputInterface $input, OutputInterface $output)
 
 	private function configureSms(InputInterface $input, OutputInterface $output) {
 		$helper = $this->getHelper('question');
-
-		$providerQuestion = new Question('Please choose a SMS provider (websms, playsms, clockworksms, puzzelsms, ecallsms, voipms, huawei_e3531, spryng, sms77io, ovh, clickatellcentral, clicksend): ', 'websms');
+		$providerArray = ['websms', 'playsms', 'clockworksms', 'puzzelsms', 'ecallsms', 'voipms', 'huawei_e3531', 'spryng', 'sms77io', 'ovh', 'clickatellcentral', 'clicksend', 'plivo'];
```

Comment:
nitpick: make this a multi line array like

```php
$providerArray = [
  'a',
  'b',
];
```

so the diff is easier to read for future additions.

Reply:
copy that.